### PR TITLE
python3Packages.python3-msrplib: init at 0.20.1

### PR DESCRIPTION
**Pull request by Yureien**
###### Description of changes

https://github.com/AGProjects/python3-msrplib/tree/master/msrplib

MSRP (RFC4975) client library

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/python3-msrplib/default.nix [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+, python3-eventlib
+, python3-gnutls
+, twisted
+}:
+
+buildPythonPackage rec {
+  pname = "python3-msrplib";
+  version = "0.20.1";
+
+  disabled = !isPy3k;
```
Comment (fabaff) [inline pkgs/development/python-modules/python3-msrplib/default.nix:16]: I suggest to limit the supported Python releases to the ones which are not EOL.

```suggestion
  disabled = pythonOlder "3.7";
```

File: pkgs/development/python-modules/python3-msrplib/default.nix [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{ stdenv
+, lib
```
Comment (fabaff) [inline pkgs/development/python-modules/python3-msrplib/default.nix:2]: `stdenv` is not used.

```suggestion
{ lib
```

File: pkgs/development/python-modules/python3-msrplib/default.nix [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+buildPythonPackage rec {
+  pname = "python3-msrplib";
+  version = "0.20.1";
+
+  disabled = !isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "AGProjects";
+    repo = pname;
+    rev = "5bd069620d436d5a65e1c369e43cc6b88857fb9e";
```
Comment (fabaff) [inline pkgs/development/python-modules/python3-msrplib/default.nix:21]: Source is [tagged](https://github.com/AGProjects/python3-msrplib/releases/tag/0.21.0).

```suggestion
    rev = "v${version}";
```

File: pkgs/development/python-modules/python3-msrplib/default.nix [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    owner = "AGProjects";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-rjkg6iRvnokNikd2UAyngKuS2tkeKmGdu4Uvw5lvcys=";
+  };
+
+  propagatedBuildInputs = [
+    python3-application
+    python3-eventlib
+    python3-gnutls
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/python3-msrplib/default.nix:27]: The names of these packages including this one is not very great. The python3 should be stripped in all of them.

File: pkgs/development/python-modules/python3-msrplib/default.nix [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    twisted
+  ];
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/python3-msrplib/default.nix:30]: Whats with the tests https://github.com/AGProjects/python3-msrplib/tree/master/test ?

**Pull request closed without merging**